spec/retry_spec.rb

```diff
@@ -295,4 +295,54 @@ def self.retry_delay(_attempt, _exception)
       end
     end
   end
+
+  describe 'retry callback' do
+    let(:retry_instance) do
+      described_class.new(strategy: :constant, callback: callback, **options)
+    end
+    let(:callback) { proc { raise 'callback executed' } }
```

Comment:
Same here, `#call` is never called by `instance_exec`. How to create a double that can be `instance_exec`ed?